extension/src/popup/components/WarningMessages/styles.scss

```diff
@@ -385,6 +385,28 @@
       line-height: 1.375rem;
       margin: 0;
     }
+
+    .CopyText {
```

Comment:
You'll notice we had to add very similar styles in several places. This is done to handle the slight differences in layout needed from the CopyText SDS component. This resulted in a change in CopyText to allow for a headless and/or block mode which will allow us to consolidate this a bit.